db/migrations/1.sql

```diff
@@ -0,0 +1,8 @@
+\c e_magnet
+ALTER TABLE magnets RENAME TO users_magnets;
+ALTER TABLE users_magnets RENAME COLUMN magnet_id TO user_magnet_id;
```

Comment:
**user_magnet_id;** why not just **user_id** ?

Reply:
because it is id of relationship